Bottom: f683ac5bfc57bf24c7837b4b92655d800937c6ba
Top:    eb29ea5cb168241e14ffca932a2bd4f5be26b986
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-11 16:45:32 +0100

Refresh of add_pours

---

diff --git a/src/mymem.h b/src/mymem.h
index 2732539..bdae9a2 100644
--- a/src/mymem.h
+++ b/src/mymem.h
@@ -57,7 +57,7 @@
 #define	STEP_REMOVELIST		500
 #define	STEP_UNDOLIST		500
 #define	STEP_POUR		10
-#define	STEP_POLYGON		10
+#define	STEP_POLYGON		50
 #define	STEP_POLYGONPOINT	10
 #define	STEP_LIBRARYMENU	10
 #define	STEP_LIBRARYENTRY	20
